vzstorage-pd/virtuozzo-provisioner.go

```diff
@@ -303,9 +303,19 @@ func (p *vzFSProvisioner) Provision(options controller.VolumeOptions) (*v1.Persi
 	storageClassOptions["volumeID"] = share
 	storageClassOptions["size"] = fmt.Sprintf("%d", bytes)
 	secretName := storageClassOptions["secretName"]
-	delete(storageClassOptions, "secretName")
+	optionsFromSystem := storageClassOptions["optionsFromSystem"]
```

Comment:
We can't guarantee that  storageClassOptions has the optionsFromSystem key, can we?

Should we do something like this?
optionsFromSystem, _ := storageClassOptions["optionsFromSystem"]

Reply:
That's right, we can't guarantee that storageClassOptions has the optionsFromSystem key, but if I'm not mistaken, in this case we would get the zero value for the type, which in this case would be "" (empty string), so the result is the same

Reply:
You are right. I messed up with python:)